src/amo/components/Addon/index.js

```diff
@@ -452,8 +453,14 @@ export class AddonBase extends React.Component {
       isRestartRequired = addon.isRestartRequired;
     }
 
+    const addonsByAuthorsLength = addonsByAuthors ? addonsByAuthors.length : 0;
+    const _classNames = classNames('Addon', `Addon-${addonType}`, {
+      'Addon--has-more-addons': addonsByAuthorsLength > 0,
+      'Addon--has-many-more-addons': addonsByAuthorsLength > 3,
```

Comment:
This is cute but not descriptive 😄

This should be `Addon--has-more-than-3-addons`. Arguable for consistency this could be:

```js
const _classNames = classNames('Addon', `Addon-${addonType}`, {
  'Addon--has-more-than-0-addons': addonsByAuthorsLength > 0,
  'Addon--has-more-than-3-addons': addonsByAuthorsLength > 3,
});
```

Sure it's kinda weird and verbose but it's obvious and consistent.

Reply:
I was not sure about it, but it sounds good to me.